sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -74,10 +74,10 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
         /// </summary>
         /// <param name="endpoint">A <see cref="Uri"/> to the service the client
         /// sends requests to.  Endpoint can be found in the Azure portal.</param>
-        /// <param name="subscriptionKey">The subscription key used to access
+        /// <param name="credential">The subscription key used to access
```

Comment:
Let's add the same sentence here as in the samples ... "Passing ... allows you to ...."